Scenarios/ITU-R_M2412-0_5BB.m

```diff
@@ -0,0 +1,51 @@
+clear all 
+
+% Disable cast to struct warnings
+w = warning ('off','all');
+
+%% Get configuration
+Config = MonsterConfig(); % Get template config parameters
+
+%add scenario specific setup for 'ITU-R M2412-0 5.B.B': 
+    % from https://www.itu.int/dms_pub/itu-r/opb/rep/R-REP-M.2412-2017-PDF-E.pdf Table 5.c Configuration B
+    %For Spectral efficiency and mobility Evaluations.
+    Config.Scenario = 'ITU-R M.2412-0 5.B.B';
+    Config.MacroEnb.number = 19;
+    Config.MicroEnb.number = 0;
+    Config.PicoEnb.number = 0;				
+    Config.Phy.downlinkFrequency = 30000; %MHz
+    Config.MacroEnb.height= 25;
+    Config.MacroEnb.Pmax = 10^(37/10)/1e3; %37dBm converted to W
+    %TODO: make 40MHz bandwidth possible
+    Config.MacroEnb.subframes = 100; % should be 200 corresponding to 40MHz bandwidth
```

Comment:
Subframes? This is number of resource blocks no?

Reply:
There are a few of these, also for the UE. Subframes is not the right terminology here. It would be resource blocks